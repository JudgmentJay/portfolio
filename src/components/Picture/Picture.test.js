import React from 'react'
import renderer from 'react-test-renderer'

import { Picture } from '../'

const mockData = {
	imgName: 'job_saatva',
	fileType: 'jpg',
	alt: 'Software Engineer II',
	widths: [460, 100, 125],
	lazyLoad: false,
	className: 'mockClass'
}

it('Renders correctly', () => {
	const tree = renderer
		.create(<Picture {...mockData} />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
