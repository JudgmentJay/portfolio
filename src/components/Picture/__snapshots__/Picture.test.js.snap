// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<picture>
  <source
    media="(max-width: 519px)"
    srcSet="img/job_saatva_m.webp"
    type="image/webp"
  />
  <source
    media="(max-width: 519px)"
    srcSet="img/job_saatva_m.jpg"
    type="image/jpeg"
  />
  <source
    media="(max-width: 739px)"
    srcSet="img/job_saatva_t.webp"
    type="image/webp"
  />
  <source
    media="(max-width: 739px)"
    srcSet="img/job_saatva_t.jpg"
    type="image/jpeg"
  />
  <source
    srcSet="img/job_saatva.webp"
    type="image/webp"
  />
  <source
    srcSet="img/job_saatva.jpg"
    type="image/jpeg"
  />
  <img
    alt="Software Engineer II"
    className="mockClass"
    sizes="(max-width: 519px) 460px, (max-width: 739px) 100px, 125px"
    src="img/job_saatva.jpg"
    srcSet="img/job_saatva.jpg 460w, img/job_saatva.jpg 100w, 125w"
  />
</picture>
`;
